### PR TITLE
Fix/edit provider and client

### PR DESCRIPTION
**Pull request by lucas-araujo-dev**
Added missing fields in the edit tab

Screenshot from 2020-10-14 21-02-55
Screenshot from 2020-10-14 21-03-31

### CODE REVIEW

File: app/views/adms/registrations/edit.html.erb [hunk 1]
```diff
@@ -13,7 +13,7 @@
   <% end %>
 
   <div class="field">
-    <%= f.label :password %> <i>(leave blank if you don't want to change it)</i><br />
+    <%= f.label :password %> <i>(leave blank if you dont want to change it)</i><br />
```
Comment (renatamarques97) [inline app/views/adms/registrations/edit.html.erb:16]: why did you change here?

Comment (renatamarques97) [reply app/views/adms/registrations/edit.html.erb:16]: don't you think that's better to use i18n?

Comment (Lucaas-Eduardo) [reply app/views/adms/registrations/edit.html.erb:16]: was reporting an error, the program recognizes it as opening quotation marks

Comment (renatamarques97) [reply app/views/adms/registrations/edit.html.erb:16]: use `do not` instead.

Comment (renatamarques97) [reply app/views/adms/registrations/edit.html.erb:16]: and use i18n.

**Review by AleBL — APPROVED** [review]
very good for me everything is fine, just add the text that @renatamarques97 mentioned in i18n and that's it
